www/main/common/sttm-ui/search-results/SearchResults.jsx

```diff
@@ -86,7 +86,7 @@ const SearchResults = ({
     <li onClick={() => onClick(shabadId, verseId)} className="search-li">
       <div className={`search-list ${getBorderColorClass(sourceId)}`}>
         <a className="panktee">
-          {ang !== null && (
+          {Boolean(ang) !== false && (
```

Comment:
Do we really need to compare `Boolean(ang) !== false &&` with `false`?

This can be refactored as `Boolean(ang) &&` or even shorter `!!ang &&`

Reply:
Good catch @inderpreetsingh updated and pushed the code